src/nvim/option_vars.h

```diff
@@ -429,6 +429,7 @@ EXTERN unsigned cb_flags;
 EXTERN OptInt p_cwh;            ///< 'cmdwinheight'
 EXTERN OptInt p_ch;             ///< 'cmdheight'
 EXTERN char *p_cms;             ///< 'commentstring'
+EXTERN int p_cp;                ///< 'compatible'
```

Comment:
It's a bit of a step backwards if we need to re-introduce dummy vars for disabled options. It would be better if it was the other way around, i e we would be able to delete `p_enc` etc while keeping the existing user-facing behavior.